Example/Kommunicate/Base.lproj/Localizable.strings

```diff
@@ -162,3 +162,18 @@ NoConversationsLabelText = "No conversations";
 UnableToCreateConversationError = "Unable to create conversation";
 OkButton = "Okay";
 WaitMessage = "Please wait...";
+
+/* Last Message of conversation */
+Photo = "photo1";
+Location = "Location1";
```

Comment:
Why are we adding `1` as suffix in all these new values?

Reply:
this was added for testing.